modules/admin_manual/pages/release_notes.adoc

```diff
@@ -33,6 +34,66 @@
+For these reasons, user-key storage encryption is now marked as deprecated and will not be maintained anymore for future versions of ownCloud Server. Server 10.7 still supports user-key encryption but the feature will be removed in later versions. If you are operating an ownCloud installation with user-key storage encryption enabled, please get in contact with support@owncloud.com to plan a migration to master-key storage encryption.
+
+TIP: Master-key storage encryption is still supported and has received improvements with Server 10.7 (see above). This encryption mode can be used with dedicated https://doc.owncloud.com/server/latest/admin_manual/configuration/server/security/hsmdaemon/[HSM products] for additional security.
+
+=== Other Notable Changes
+
+* Redis can now be connected with TLS support for improved security. See the https://doc.owncloud.com/server/admin_manual/configuration/server/config_sample_php_parameters.html#define-redis-connection-details[documentation] for more information. https://github.com/owncloud/core/pull/38386[#38386]
+* For strong security, ownCloud Server uses strict same-site cookie handling. In certain scenarios (e.g., integrations) this behavior is not desired. To be able to flexibly adapt the intended behavior, https://doc.owncloud.com/server/latest/admin_manual/configuration/server/config_sample_php_parameters.html#define-how-to-relax-same-site-cookie-settings[the same-site cookie handling can now be configured]. https://github.com/owncloud/core/pull/38458[#38458]
+* Loading the "Shared with you" list when shares originate from files on unavailable storages (e.g., Windows Network Drives) has been fixed. https://github.com/owncloud/core/pull/38190[#38190]
+* Performance improvements for the "Shared with you" view list have been made. https://github.com/owncloud/core/pull/38385[#38385]
+* Existing guest users are now correctly labeled as 'Guest' in the sharing sidebar tab (before they were labeled as regular 'User'). https://github.com/owncloud/core/pull/38440[#38440]
+* Issues with multiple files with the same name e.g., in the "Shared by link" view, have been fixed https://github.com/owncloud/core/pull/38415[#38415]
+* Error messages when uploading files are more specific again (e.g., when a virus has been found or when an upload was blocked due to a File Firewall rule) https://github.com/owncloud/core/pull/38416[#38416]
+* The https://doc.owncloud.com/server/admin_manual/configuration/server/occ_command.html#mimetype-update-commands[occ command] `maintenance:mimetype:update-db --repair-filecache` has been fixed. It can be used to bring changed mimetype associations for files into operation. https://github.com/owncloud/core/issues/38425[#38425]
```

Comment:
```suggestion
* The xref:configuration/server/occ_command.adoc#mimetype-update-commands[occ command] `maintenance:mimetype:update-db --repair-filecache` has been fixed. It can be used to bring changed mimetype associations for files into operation. https://github.com/owncloud/core/issues/38425[#38425]
```